scripts/generate-k8s-yaml

```diff
@@ -47,20 +47,35 @@ while getopts "vn:" opt; do
   esac
 done
 
-curl -L https://get.helm.sh/helm-v$HELM_VERSION-$PLATFORM-amd64.tar.gz | tar zxf - -C $BUILD_DIR
-mv $BUILD_DIR/$PLATFORM-amd64/helm $BUILD_DIR/.
-rm -rf $BUILD_DIR/$PLATFORM-amd64
-chmod +x $BUILD_DIR/helm
+INDV_RESOURCES_DIR=${BUILD_DIR}/individual-resources${SUFFIX}
+TAR_RESOURCES_FILE=${BUILD_DIR}/individual-resources${SUFFIX}.tar
+AGG_RESOURCES_YAML=${BUILD_DIR}/all-resources${SUFFIX}.yaml
+mkdir -p $INDV_RESOURCES_DIR
+
+QP_INDV_RESOURCES_DIR=${BUILD_DIR}/individual-resources-queue-processor${SUFFIX}
+QP_TAR_RESOURCES_FILE=${BUILD_DIR}/individual-resources-queue-processor${SUFFIX}.tar
+QP_AGG_RESOURCES_YAML=${BUILD_DIR}/all-resources-queue-processor${SUFFIX}.yaml
+mkdir -p $QP_INDV_RESOURCES_DIR
+
+HELM=$BUILD_DIR/helm
+if [[ ! -e $HELM ]]; then
```

Comment:
what's the purpose of this check?

Reply:
The script downloads its own copy of the helm executable if doesn't already exist in the build directory. Since we know the script will be run twice this check prevents an unnecessary re-download.